scripts/deploy-code.sh

```diff
@@ -42,9 +42,9 @@ echo "Performing an install to make sure that all dependencies are installed"
 yarn install --frozen-lockfile
 echo "yarn install done."
 
-echo "Compiling typescript code so it's ready to be uploaded"
-yarn typescript
-echo "compiling code done"
+echo "Running pre-deploy script to compile code"
+yarn pre-deploy
+echo "pre-deploy step complete"
```

Comment:
Made this change so that the `dev:update` script and this deployment script run the same commands for compiling the RN SDK. This helps improve our QA testing by installing the local SDK into sample apps the same way that customers install our SDK in production.